web/source/osk/visualKeyboard.ts

```diff
@@ -1491,17 +1518,29 @@ namespace com.keyman.osk {
       kbd.style.border = '1px solid #ccc';
 
       // Once the element is inserted into the DOM, refresh the layout so that proper text scaling may apply.
-      const refreshInterval = window.setInterval(function () {
-        let computedStyle = getComputedStyle(kbd);
-        if (computedStyle.fontSize) {
-          if (kbd.style.fontSize) {
-            // Preserve the new setting (provided by CSS)
-            kbdObj.fontSize = new ParsedLengthStyle(kbd.style.fontSize);
+      const detectAndHandleInsertion = () => {
```

Comment:
I remember a comment or two from recent reviews of other PRs that the use of a timeout here is somewhat suboptimal.  After a bit of thought and brainstorming, I realized the same goals could be handled via `MutationObserver` instead.